huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go

```diff
@@ -190,7 +195,7 @@ func resourceDmsRocketMQConsumerGroupUpdate(ctx context.Context, d *schema.Resou
 	return resourceDmsRocketMQConsumerGroupRead(ctx, d, meta)
 }
 
-func buildUpdateRocketmqConsumerGroupBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateRocketmqConsumerGroupBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```

Comment:
If the config is not used, we should remove it from the arguemnt list.

Reply:
done